.gitmodules

```diff
@@ -24,4 +24,7 @@
 	url = https://github.com/XRTK/SpatialPersistence.git
 [submodule "Submodules/SpatialPersistence.ASA"]
 	path = Submodules/SpatialPersistence.ASA
-	url = https://github.com/XRTK/SpatialPersistence.ASA.git
\ No newline at end of file
+	url = https://github.com/XRTK/SpatialPersistence.ASA.git
+[submodule "Submodules/Pico"]
```

Comment:
Let's add this in it's own specific PR

Reply:
This is intended to be removed before finishing draft.